### ISSUE TITLE
Distribute .d.ts typings along with the package

### ISSUE THREAD

**Issue by grrowl** [fabienjuif/use-bus#11]
There seems to be .d.ts typings in the repository but they're missing in the dist package itself:

```
node_modules/use-bus/
├── LICENSE
├── README.md
├── dist-node
│   ├── index.js
│   └── index.js.map
├── dist-src
│   ├── index.js
│   └── index.spec.js
├── dist-web
│   ├── index.js
│   └── index.js.map
└── package.json
```

They should be included alongside the .js files in all environments

**Comment by fabienjuif** [comment]
Hello @grrowl 👋 

I'll look after this today if I can

**Comment by fabienjuif** [comment]
@grrowl this is published in 2.3.1 version.
Feel free to open issues and/or PR!

Cya,
